packages/dashboard/src/components/drawers/tests/drawer-header.test.tsx

```diff
@@ -1,49 +1,44 @@
 import React from 'react';
-import { createShallow } from '@material-ui/core/test-utils';
+import { render } from '@testing-library/react';
 import DrawerHeader from '../drawer-header';
-import CloseIcon from '@material-ui/icons/Close';
-import { KeyboardBackspace as BackIcon } from '@material-ui/icons';
-import { Typography } from '@material-ui/core';
-
-const mount = createShallow();
 
 describe('Drawer header', () => {
   test('Renders correctly', () => {
-    const root = mount(
+    const root = render(
       <DrawerHeader
         handleCloseButton={() => jest.fn()}
         handleBackButton={() => jest.fn()}
         title={'Test'}
       />,
     );
-    expect(root.find(Typography).html()).toContain('Test');
-    expect(root.find(CloseIcon).exists()).toBeTruthy();
-    expect(root.find(BackIcon).exists()).toBeTruthy();
-    expect(root).toMatchSnapshot();
+    expect(root.getByText('Test')).toBeDefined();
+    expect(root.getByTestId('backDrawerButton')).toBeDefined();
+    expect(root.getByTestId('closeDrawerButton')).toBeDefined();
+    //expect(root).toMatchSnapshot();
   });
 
   test('Renders without back button if there is no handler', () => {
-    const root = mount(<DrawerHeader handleCloseButton={() => jest.fn()} title={'Test'} />);
-    expect(root.find(BackIcon).exists()).toBeFalsy();
+    const root = render(<DrawerHeader handleCloseButton={() => jest.fn()} title={'Test'} />);
+    expect(root.queryByTestId('backDrawerButton')).toBeFalsy();
   });
 
   test('Click on close button fires an event correctly', () => {
     let clicked = false;
-    const root = mount(<DrawerHeader handleCloseButton={() => (clicked = true)} title={'Test'} />);
-    root.find('#closeDrawerButton').simulate('click');
+    const root = render(<DrawerHeader handleCloseButton={() => (clicked = true)} title={'Test'} />);
+    root.getByTestId('closeDrawerButton').click();
```

Comment:
I'm curious, Is this working? with the click in the end?

Reply:
yea it actually does but it uses the [web API](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/click).. so I'll probably change this to userEvent instead, as per your previous comments?

putting a console log line before and after the clicks to log the variable 'clicked':
```
console.log
    false
      at Object.<anonymous> (src/components/drawers/tests/drawer-header.test.tsx:41:13)
  console.log
    true
      at Object.<anonymous> (src/components/drawers/tests/drawer-header.test.tsx:43:13)
```